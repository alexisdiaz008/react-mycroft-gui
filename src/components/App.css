body {
	background-color: black;
}

.App {
	height: 100%;
	/*padding: 14rem 0;
  width: 100%;*/
} 