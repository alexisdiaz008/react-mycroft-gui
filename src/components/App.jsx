import React, { Component } from 'react';
import Face from './face/face'
import MycroftMessageBusSkillComponent from './mycroft_message_bus_elements/mycroft_message_bus';
import 'bootstrap/dist/css/bootstrap.min.css';
import './app.css';

class App extends Component {	
  render() {
    return (
      <div className="App container">
        <Face />
        <MycroftMessageBusSkillComponent />
      </div>
    );
  }
}

export default App;
