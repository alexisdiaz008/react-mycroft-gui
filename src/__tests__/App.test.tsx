import * as React from 'react';
import { shallow } from 'enzyme';
import App from '../components/App';


describe('<App />', () => {
  test('renders the component', () => {
    const component = shallow(<App />);  expect(component).toMatchSnapshot();
  });
});
