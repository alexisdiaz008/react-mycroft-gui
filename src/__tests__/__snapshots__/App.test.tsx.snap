// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders the component 1`] = `ShallowWrapper {}`;
